includes/Story_Renderer/HTML.php

```diff
@@ -215,6 +216,23 @@ protected function replace_html_head( $content ) {
 		return $content;
 	}
 
+	/**
+	 * Force home urls to http / https based on context.
+	 *
+	 * @param string $content String to replace.
+	 *
+	 * @return string
+	 */
+	protected function replace_url_scheme( $content ) {
+		$home           = get_home_url();
```

Comment:
```suggestion
		$home           = home_url();
```

With that said, `home_url()` supports a `$scheme` param...